### PR TITLE
Issue 80 - Handling Out of Bound Errors for Corgme

### LINKED ISSUE #80: Corgme no longer correctly handles index out of bounds errors
**Describe the bug**
Bot produced an unexpected exception when attempting to fetch a very large numbered corgi. Exception was

```
Unexpected error: Command raised an exception: IndexError: list index out of range
```

**To Reproduce**
1. Run `-corgme 245634567` (or any other very large number)
2. Bot throws an unexpected error

**Expected behavior**
Either a graceful failure message, a random corgi, or both

**Screenshots**
image

**Discord Link**
[link](https://discord.com/channels/687722620572925986/799305096369537084/868853978576859156)

### PR DESCRIPTION
**Pull request by mbozada**
# Description

I added an upper bound to the corgme command. Now, if the input number is greater than 99, a random corgi is displayed.

## Issues

Closes #80 

## Type of change

Select one or more of the following:

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other (describe below)

# How Has This Been Tested?

- [x] -corgme -1
- [x] -corgme 0
- [x] -corgme 99
- [x] -corgme 100
- [x] -corgme 2000

# Checklist:

- [x] All local commits have been pushed to remote
- [x] All changes on the base branch have been merged into this branch, either by rebase or merge
- [x] My code is [PEP-8](https://pep8.org/) compliant (excluding maximum line length, keep that to 100ish characters)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added [Google Docstrings](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) for all new functions/methods
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by tux2603** [comment]
Would it be better to check if the number is greater than the length of the image array? With this approach we'll have corgi images that are never displayed

**Comment by CodingPenguin1** [comment]
> Would it be better to check if the number is greater than the length of the image array? With this approach we'll have corgi images that are never displayed

Agreed. 99 is arbitrary

File: Cogs/StudentCommands.py [hunk 1]
```diff
@@ -42,7 +42,7 @@ async def corgme(self, ctx, number=-1):
         images = ['dogs/corgis/' + path.name for path in Path('dogs').rglob('*.*')]
 
         # Generates a random number if no number is given
-        if number < 0:
+        if number < 0 or number > 99:
```
Comment (tux2603) [inline Cogs/StudentCommands.py:45]: This should be changed to `number > len(images)`

**Review by tux2603 — CHANGES_REQUESTED** [review]

File: Cogs/StudentCommands.py (hunk 1 above)
Comment (mbozada) [reply Cogs/StudentCommands.py]: Fixed with new commit.

**Review by CodingPenguin1 — APPROVED** [review]

**Review by tux2603 — APPROVED** [review]

**Pull request merged**